server/routes/article.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const router = express.Router();
+const articleLibs = require('../libs/articlelibs');
+const {authenticate} = require('../middleware/authenticate');
+
+router.post('/article/', (req, res) => {
+    let data = {
+        topic : req.body.topic,
+        article : req.body.article,
+        uuid : req.header('uuid')
+    }
+    articleLibs.createArticle(data).then((article) => {
+        res.status(article.status).send(article);
+    }).catch((error) => {
+        res.status(error.status).send(error);
+    });
+});
+
+router.get('/article/:articleId', (req, res) => {
+    let query = {
+        articleId : req.params.articleId
+    }
+    articleLibs.getArticle(query).then((article) => {
+        res.status(article.status).send(article);
+    }).catch((error) => {
+        res.status(error.status).send(error);
+    });
+});
+
+router.get('/article/:topic', (req, res) => {
+    let query = {
```

Comment:
not need to create a variable for single field in object just do articleLibs.getArticle({topic : req.params.topic})

Reply:
Done